app/assets/javascripts/components/features/ui/components/column_link.jsx

```diff
@@ -1,7 +1,6 @@
 import { Link } from 'react-router';
 
 const outerStyle = {
-  display: 'block',
```

Comment:
This is what I wasn't clear on; I don't know why the style is inline but I had to move the `display` property to CSS to be able to override it from within the stylesheet. If more/all of these should be moved over, or if there's a better way to do any of this, let me know 👍